tests/baselines/reference/getParameterNameAtPosition.types

```diff
@@ -23,7 +23,7 @@ declare function fn<Y extends any[]>(implementation?: (...args: Y) => any): Mock
 cases(fn(opts => { }));
 >cases(fn(opts => { })) : void
 >cases : (tester: Tester) => void
->fn(opts => { }) : Mock<[opts: any]>
+>fn(opts => { }) : Mock<[opts: any, done: (...args: any[]) => any]>

```

Comment:
This looks correct to me; `Tester` has this parameter but it was dropped because the arrow function didn't declare the parameter.